### PR TITLE
remove context namedtuple.

### PR DESCRIPTION
**Pull request by junchaowu**
@blampe @breerly @abhinav 
remove the context(message, message_id) namedtuple.
Increase the TTL for request

### CODE REVIEW

File: python/tchannel/messages/common.py [hunk 1]
```diff
@@ -33,6 +33,8 @@
 
 PROTOCOL_VERSION = 0x02
 
+DEFAULT_TTL = 1000  # ms
```
Comment (breerly) [inline python/tchannel/messages/common.py]: Shouldn't this be in `glossary`?

File: python/tchannel/handler.py [hunk 2]
```diff
@@ -108,17 +104,17 @@ def handle_pre_call(self, message_id, message, connection):
...
-                message_id,
+                message.id,
             )
-        except Exception:
+        except Exception as e:
             connection.send_error(
                 ErrorCode.unexpected,
-                "An unexpected error has occurred!",
-                message_id,
+                "An unexpected error has occurred!" + e.message,
```
Comment (abhinav) [inline python/tchannel/handler.py]: Probably not a good idea to include service-internal error messages in external responses. Skip doing this. Also, drop the `!`.

(If we were to include the message, concatenation like that doesn't make for a pretty error message.)

File: python/tchannel/messages/error.py [hunk 3]
```diff
@@ -47,6 +47,7 @@ class ErrorMessage(BaseMessage):
     message_type = Types.ERROR
 
     __slots__ = (
+        'id',
```
Comment (abhinav) [inline python/tchannel/messages/error.py]: `BaseMessage.__slots__`?

File: python/tchannel/messages/init_request.py [hunk 4]
```diff
@@ -38,7 +38,8 @@ class InitRequestMessage(BaseMessage):
         'headers',
     )
 
-    def __init__(self, version=None, headers=None):
+    def __init__(self, version=None, headers=None, id=0):
+        super(InitRequestMessage, self).__init__(id)
```
Comment (abhinav) [inline python/tchannel/messages/init_request.py]: `BaseMessage.__slots__`?

Comment (junchaowu) [inline python/tchannel/messages/init_request.py]: done

**Comment by abhinav** [comment]
:ok:

**Pull request merged**